README.md

```diff
@@ -53,6 +53,15 @@ Capital Framework is our modular front end framework. New projects should start
 - **Willingness**: We are willing to help each other with anything that is good for the team
 - **Excellence**: We believe that excellence is defined by a commitment to and practice of constant learning and improvement
 
+## Our Front End Web Development Values
+
+1. End goals of the user should be what drive the front-end design.
+2. Pages, apps, and sites should be inclusive of all users.
+3. We want to continue to use cutting-edge technologies and methods to build modern applications, but we won’t allow that to preclude usability for all consumers.
```

Comment:
Usability it not something that you can preclude. How about "… won't allow that to negatively impact usability …"?